drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -501,9 +583,73 @@ local function occupancy_attr_handler(driver, device, ib, response)
   device:emit_event(ib.data.value == 0x01 and capabilities.motionSensor.motion.active() or capabilities.motionSensor.motion.inactive())
 end
 
+local function power_topology_handler(driver, device, ib, response)
```

Comment:
I believe this should just give us the value of the power topology for the given endpoint that this message was received on, so we should just be able to use the `ib.data` to get the topology type rather than having to check for feature maps on the endpoints.
```
device:set_field(POWER_TOPOLOGY_TYPE, ib.data)
```

Reply:
Hm, what would `ib.data` be returning from this def: [clusters.PowerTopology.ID]` = power_topology_handler,`?

Reply:
Either way, I don't think we actually need this handling. The way we are using the device type as of now, there is no reason to support power topology.